sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_critical_result_inference_async.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import asyncio
+import datetime
+import os
+
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights.aio import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
```

Comment:
If you move these inside the sample code, customers will more easily see how to import the client, models, etc

Reply:
done